disnake/permissions.py

```diff
@@ -198,9 +213,24 @@ def all_channel(cls: Type[P]) -> P:
         .. versionchanged:: 2.3
             Added :attr:`start_embedded_activities` permission.
         """
-        return cls(0b1111110110110011111101111111111101010001)
+        guild_specific_perms = {
+            "administrator",
+            "ban_members",
+            "change_nickname",
+            "kick_members",
+            "manage_emojis",
```

Comment:
```suggestion
            "manage_emojis",
            "manage_events",
```
(also missing from the docstring)

Reply:
manage events is a per-channel permission.